lib/ae_mdw/sync/server.ex

```diff
@@ -298,15 +299,29 @@ defmodule AeMdw.Sync.Server do
     new_state =
       gens_mutations
       |> Enum.flat_map(fn {_height, blocks_mutations} -> blocks_mutations end)
-      |> Enum.reduce(state, fn {_block_index, _block, block_mutations}, state ->
-        State.commit_db(state, block_mutations, clear_mem?)
+      |> Enum.reduce(state, fn {block_index, block, block_mutations}, state ->
+        state
+        |> maybe_enqueue_accounts_balance(block, block_index)
+        |> State.commit_db(block_mutations, clear_mem?)
       end)
 
     broadcast_blocks(gens_mutations)
 
     new_state
   end
 
+  defp maybe_enqueue_accounts_balance(state, _block, {_kbi, -1}) do
```

Comment:
Key blocks would affect miner's balances, would it make sense to include them too?

Reply:
Yes, gonna be implemented incrementally. Next there will be these items from the Notes:
- the config of amount of wallets
- consider internal transfers and rewards
- automated tests of async tasks

Reply:
Cool, thanks for explaining